pom.xml

```diff
@@ -266,8 +266,8 @@
         <httpclient.version>4.5.6</httpclient.version>
         <jctools.version>2.0.1</jctools.version>
         <jgrapht.version>0.9.0</jgrapht.version>
-        <guava.version>16.0.1</guava.version>
-        <auto-service.version>1.0-rc3</auto-service.version>
+        <guava.version>27.0.1-jre</guava.version>
+        <auto-service.version>1.0-rc4</auto-service.version>
```

Comment:
Just curious, why is the version changed for `auto-service` here? Is it related to guava?

Reply:
Yes. Rc3 uses Guava 19, while 4 uses 23.5. I think as of Guava 21, there have been no breaking changes going forward.

Reply:
OK makes sense.